net/strongswan/files/swanctl.init

```diff
@@ -227,26 +227,34 @@ config_child() {
 	local updown
 	local firewall
 	local remote_subnet
-	local lifetime
+	local life_time
 	local dpdaction
 	local closeaction
 	local startaction
 	local if_id
-	local rekeytime
+	local rekey_time
+	local rekey_bytes
+	local life_bytes
+	local rekey_packets
+	local life_packets
 
 	config_get startaction "$1" startaction "route"
 	config_get local_nat "$1" local_nat ""
 	config_get updown "$1" updown ""
 	config_get firewall "$1" firewall ""
-	config_get lifetime "$1" lifetime ""
+	config_get life_time "$1" lifetime ""
```

Comment:
Changing the names of the settings breaks backwards compatibility.  Please don't do this.

Reply:
Hey @pprindeville - I have tested this with my existing configuration using `lifetime`, and AFAICS the first argument to `config_get` is the name of the bash local, not the name of the variable in the config. Therefore this doesn't break backwards compatibility.

The naming is rather inconsistent in this function, and so I can't figure out if just `concatenatingallthewords` is preferred, or if using the swanctl.conf `snake_case` keyword is preferred. The `/etc/config/ipsec` keys are all still `singleword`.

Reply:
Using single words is preferred, unless you're willing to write a filter that handles the up-conversion...

Reply:
@pprindeville fixed and rebased against master. Please have a look.